tests/ReifiedGenericsTest.hack

```diff
@@ -75,4 +75,25 @@ final class ReifiedGenericsTest extends \Facebook\HackTest\HackTest {
         ->toString(),
     )->toEqual('HH\dict<string, ?HH\dict<string, ?HH\dict<string, null>>>');
   }
+
+  public function testNonNullType(): void {
+    expect(TypeSpec\of<nonnull>()->toString())->toEqual('nonnull');
```

Comment:
Can you add expectations for, assertion and coercion (i'm assuming coercion would == assertion here, except for the kind of exception) - both good and bad cases

Reply:
I think I don't understand how our tests are structured.
Did I put the wrong type of test here?

Do the `NullSpecTest` and `NonNullSpecTest` not cover the assertion aspects of this?
They have bad examples and good examples with getValidCoercions() and getInvalidCoercions().
I intended to test `TestSpec\of<T>()` here, since that effectively tests `TypeAssert\matches<T>()` and `TypeCoerce\match<T>()`, since they both use the same mechanism.

Reply:
Sorry ,the spec tests cover it, I didn't notice them :(